data/core/macros/abilities.cfg

```diff
@@ -534,6 +534,19 @@ Enemy units cannot see this unit while it is in deep water, except if they have
     [/damage]
 #enddef
 
+#define WEAPON_SPECIAL_ABSORB AMT
+    # Canned definition of the Absorb ability to be included in a
+    # [specials] clause.
+    [damage]
+        id=absorb
+        name= _ "absorb"
+        description= _ "This attack reduces the damage suffered from an enemy strike."
+        special_note= _ "This attack puts the unit in good defensive position, and it absorbs some of the damage dealt by an enemy strike."
```

Comment:
This special note seems like it might be better as the description, and the special note should be written from the perspective of the unit rather than the attack.

Reply:
OK.  The special notes could probably be completely removed, but that's out of scope here.